src/features/landing/components/Hero/HeroSection.tsx

```
@@ -7,7 +7,7 @@ import { HeroSearchForm } from './HomeSearchForm';
 export function HeroSection() {
   return (
     <section className="relative">
-      <div className="absolute inset-0 bg-gradient-radial from-[color:var(--color-accent)]/5 to-transparent opacity-70 animate-pulse-slow"></div>
+      <div className="absolute inset-0 bg-gradient-radial from-[color:var(--color-accent)]/5 to-transparent opacity-70"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)